measure_cocoapod_size.py

```diff
@@ -51,7 +51,7 @@ def GetSampleApp(mode):
     return OBJC_APP_DIR, OBJC_APP_NAME
 
 
-def InstallPods(cocoapods, target_dir, spec_repos, target_name, mode):
+def InstallPods(cocoapods, target_dir, spec_repos, target_name, mode, platform):
```

Comment:
Consider making the argument optional and `ios` the default value of platform so that existing users see no change.